### DISCUSSION TITLE
Stint Window

### DISCUSSION THREAD

**Discussion by jtokash** [f1multiviewer/issue-tracker#77] [Feature requests]
A stint window, focused on tire management and tire strategy.  Similar to this small portion of the data channel (see attachment)

<img width="437" alt="CleanShot 2022-06-28 at 15 11 49@2x" src="https://user-images.githubusercontent.com/614952/176313490-1a701ca2-d524-4d44-a646-f440b9379963.png">

**Comment by JJWatMyself** [comment]
It would be great to see the driver stats evolve. I think this would be a great addition to the existing driver stat pop-out, and have this be alongside the other racer stats.

The example below also puts things in s single row and re-orders, The current double stacked view of stats is difficult to scan vertically between the drivers for things like gap, etc.  Adding the tire data on the right and showing the tire stint becomes a natural extension of the row for that driver.

Ideally a given stat window would be 16:9 so that it can be arranged using the same principles as the other windows, or be on a dedicated screen as full screen. In these cases, the stat would need to resize / scale in order to preserve the look, irrespective of the size of the fixed aspect ratio window.

There are also some comments on the driver stat window over here: https://github.com/f1multiviewer/issue-tracker/discussions/67

![Untitled](https://user-images.githubusercontent.com/22590445/176336314-bd33b3b9-a9b2-42f7-b0a3-dda75c78cf48.jpg)

**Reply by jtokash** [reply]
That does look good, but I would mostly like a tires-only view.  Maybe everything could be toggled via settings.  The reason I'd like a tires only view is that it is sometimes hard to read when everything else is factored in.

**Comment by JJWatMyself** [comment]
Concept of single row for lap time and gap info along with tire stint is also extendable to the OBC overlay...
![image](https://user-images.githubusercontent.com/22590445/177052614-cb9027e2-592c-476c-83e3-afa4cd863ab8.png)

**Comment by f1multiviewer** [comment]
Not really done yet, but at least stints are now visible when you click on the tire compound!

**Reply by KHILL3000** [reply]
Could we get a column for "Lap started" (or something similar) for each stint, so that we can know when the tire was changed without having to do the math of Stint 1 (Total age - starting age) + Stint 2 (total age - starting age) etc.
